lui.el

```diff
@@ -168,6 +168,11 @@ changes by earlier ones."
                         (face :tag "Face"))))
   :group 'lui)
 
+(defface lui-prompt-face
+  '((t (:inherit bold)))
+  "Face used by the message prompt."
+  :group 'lui-irc-colors)
+
```

Comment:
In the past, we've refrained from simply setting a single face to the whole prompt because that means you can't style parts of the prompt individually. Why is this a good idea now?

Reply:
You are right, I can set the prompt I want with my own faces directly. I find it a bit strange though that each application has to define its own prompt. Why isn't there a lui-prompt variable that would be the default for all applications?